### DISCUSSION TITLE
Can we make items show up with zoom farther out?

### DISCUSSION THREAD

**Discussion by okdistribute** [digidem/mapeo-desktop#551] [Ideas]

**Comment by hackergrrl** [comment]
It's possible, but I think @gmaclennan capped it because performance starts to chug with too much SVG on screen at once.

**Comment by okdistribute** [comment]
It was really confusing for a user at gaia amazonas today, he thought that the data wasn't loaded correctly!

**Comment by hackergrrl** [comment]
Yeah it's really hard to find data. It was the motivation for the Zoom to Data feature actually! Once we get vector tiles working and a mapbox-gl layer in iD I think we'll be able to rig it up so we can see data from all zoom levels.

**Comment by okdistribute** [comment]
See digidem/mapeo-print-preview for details
